kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/ModemMonitorListener.java

```diff
@@ -16,5 +16,5 @@
 @ConsumerType
 public interface ModemMonitorListener {
 
-    public void setCellularSignalLevel(int signalLevel);
+    public void setCellularSignalLevel(String interfaceName, int signalLevel);
```

Comment:
I believe this requires an `@since 2.0` due to the fact that is a breaking change

Reply:
👍